renderer/pages/main.tsx

```diff
@@ -1,113 +1,194 @@
-import { Button, Checkbox, Divider, Flex, FormControl, FormLabel, Input, Table, TableCaption, TableContainer, Tbody, Td, Text, Th, Thead, Tr } from "@chakra-ui/react"
-import { useEffect, useState } from "react"
-import BaseLayout from "../components/BaseLayout"
-import electron from "electron"
-import _ from "lodash"
+import {
+  Button,
+  Checkbox,
+  Divider,
+  Flex,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  Table,
+  TableCaption,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+import BaseLayout from "../components/BaseLayout";
+import electron, { clipboard } from "electron";
+import _ from "lodash";
 
-export const PasswordView = ({ name, value, setPassList }) => {
-    const [show, setShow] = useState(false)
+export const PasswordView = ({ label, name, value, setPassList }) => {
```

Comment:
Accept UUID as a param and pass it to "pass-delete" instead of the name, since you changed the handler on ipcEvents.